server/database/config/build.sql

```diff
@@ -1 +1,73 @@
- 
+BEGIN;
+
+DROP TABLE IF EXISTS 'Users', 'Projects', 'Sections', 'Tasks', 'User_Tasks', 'User_Projects', 'Activity' CASCADE;
```

Comment:
Make all tables' names lowercase, please!
One reason for using lower case convention is so that queries are more readable with lowercase table and column names and upper case SQL keywords.